[326] Configurable preview route (#330)
github/view_component: README.md

config.view_component.preview_path = "#{Rails.root}/lib/component_previews"
```

> Previews are served from <http://localhost:3000/rails/view_components> by default. If you want to use a different endpoint you can configure the `preview_route` option:

Reviewer: ```suggestion
Previews are served from <http://localhost:3000/rails/view_components> by default. To use a different endpoint, set the `preview_route` option:
```

`config/application.rb`
```ruby

> config.view_component.preview_route = "/previews"

Reviewer 2: I think we'd call this `preview_path` ?

Author: We already have that config for enabling users configure where the previews live in the codebase. Should we rename?

Reviewer 2: oh right. hmm... I'd say `preview_source_path` and `preview_path` but that's a confusing breaking change. Let's debate the mount idea before we are forced to decide on this.

Reviewer: I prefer this approach to the mounted engine implementation. I also think `preview_route` is fine 👍
